pkg/daemon/ceph/client/filesystem.go

```diff
@@ -130,13 +123,13 @@ func CreateFilesystem(context *clusterd.Context, clusterInfo *ClusterInfo, name,
 	logger.Infof("creating filesystem %q with metadata pool %q and data pools %v", name, metadataPool, dataPools)
 	var err error
 
-	if IsMultiFSEnabled() {
+	// Always enable multiple fs when running on Pacific
+	if clusterInfo.CephVersion.IsAtLeastPacific() {
 		// enable multiple file systems in case this is not the first
 		args := []string{"fs", "flag", "set", "enable_multiple", "true", confirmFlag}
```

Comment:
This flag is still needed in pacific? I thought it was just because of experimental mode.

Reply:
It's needed for upgrade clusters. Newly deployed ones don't need it, actually, we can do that during the upgrade, let me change that!